GitHubAPI.py

```diff
@@ -0,0 +1,130 @@
+import requests
+import bs4
+import json
+import JIRAloading as JL
+
+issues = JL.JIRALoader('../testJIRA.xml')
+
+# Getting the issues loaded into the JIRALoader object
+issues.get_issue_data()
+
+REPO_OWNER = 'iansincolorado'
+REPO_NAME = 'pythonapis'
+
+# Only way of authenticating at the current moment is with GitHub PAT tokens, which are deleted from your GitHub account if it is found in a public repository
+PAT = ''
```

Comment:
You could set the PAT as an environmental variable or prompt for it and throw an error if it is not set.